dmscripts/generate_supplier_user_csv.py

```diff
@@ -146,6 +144,10 @@ def generate_csv_and_upload_to_s3(
     if not download_filename:
```

Comment:
We can get rid of this line now, then - `generate_user_csv` and `generate_supplier_csv` will always return a filename.